.github/draft-release-notes-config.yml

```diff
@@ -17,24 +17,35 @@ categories:
   - title: 'Breaking Changes'
     labels:
       - 'Breaking Changes'
+  - title: 'Security'
+    labels:
+      - 'Security'
+      - 'security fix'
+      - 'security vulnerability'
   - title: 'Features'
     labels:
       - 'Features'
   - title: 'Enhancements'
     labels:
       - 'Enhancements'
+      - 'enhancement'
```

Comment:
I don't think labels should be differentiated by letter case; everything should be in lower case.